docs/wiki/00 - SERVER INSTALL AND CONFIGURATION/02 - Website Configuration/Handling Large File Uploads.md

```diff
@@ -0,0 +1,14 @@
+# Handling Large File Uploads
+
+In order for users to upload large files (> 2 MB), you'll need to configure
+Apache's `php.ini` configuration file. 
+
+Recommended sample values appropriate for compressed scans not exceeding 500M in size are: 
```

Comment:
```suggestion
Recommended sample values appropriate for compressed scans not exceeding 500MB in size are: 
```

Reply:
@johnsaigle ping.. I tried to click "Commit" on github so I can merge this PR but I don't have permission on the branch.